### PR TITLE
Cloud Foundation: fix networkPolicy type in GKE schema file

### PR DESCRIPTION
**Pull request by claudiobizzotto**

### CODE REVIEW

**Comment by claudiobizzotto** [comment]
Any news on this one?

**Comment by ocsig** [comment]
Thank you for your contribution and pointing out this error. We are fixing it here and in the new repo. I encourage you to use the new repo:
https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/372

File: community/cloud-foundation/templates/gke/gke.py.schema [hunk 1]
```diff
@@ -328,7 +328,7 @@ properties:
           https://kubernetes.io/docs/concepts/services-networking/networkpolicies/
         properties:
           provider:
-            type: array
+            type: string
             description: The selected network policy provider.
             items:
```
Comment (sjvanrossum) [inline community/cloud-foundation/templates/gke/gke.py.schema]:
```suggestion
            enum:
              - PROVIDER_UNSPECIFIED
              - CALICO
```

File: community/cloud-foundation/templates/gke/gke.py.schema [hunk 2]
```diff
@@ -328,7 +328,7 @@ properties:
...
               type: string
```
Comment (sjvanrossum) [inline community/cloud-foundation/templates/gke/gke.py.schema]:
```suggestion
```

File: community/cloud-foundation/templates/gke/gke.py.schema [hunk 3]
```diff
@@ -328,7 +328,7 @@ properties:
           https://kubernetes.io/docs/concepts/services-networking/networkpolicies/
         properties:
           provider:
-            type: array
+            type: string
```
Comment (sjvanrossum) [inline community/cloud-foundation/templates/gke/gke.py.schema]:
```suggestion
             type: string
             enum:
               - PROVIDER_UNSPECIFIED
               - CALICO
```

**Comment by sjvanrossum** [comment]
Fixed the enum declaration, looks good now.
Thank you for this!

**Pull request merged**